benchmark/metrics/logging_metrics_benchmark.cpp

```diff
@@ -9,7 +9,7 @@
 #include "storage/storage_defs.h"
 #include "storage/write_ahead_log/log_manager.h"
 
-#define LOG_FILE_NAME "/mnt/ramdisk/benchmark.txt"
+#define LOG_FILE_NAME "benchmark.txt"
```

Comment:
@17zhangw I in fact changed the logging dir for `LoggingMetricsBenchmark` from RAM disk to real disk to run experiments locally. You just need to change it back...

Reply:
Noticed and reverted!